test/models/article_test.rb

```diff
@@ -3,9 +3,14 @@
 require 'test_helper'
 
 class ArticleTest < ActiveSupport::TestCase
-  test 'thumbnail_url' do
+  test 'prepared_thumbnail_url' do
```

Comment:
```suggestion
  test '#prepared_thumbnail_url' do
```

他に合わせてこう言う表記が良さそうです。

Reply:
@komagata 
メソッド名をテスト名にする場合は`#`をつけるという認識であっておりますか。

Reply:
@Sasaco-JM 

> メソッド名をテスト名にする場合は#をつけるという認識であっておりますか。

YESかNOかで言えばNOです。

オブジェクト指向言語の監修で、

`クラス名#インスタンスメソッド()`
`クラス名.クラスメソッド()

という記法があります。
ですのでインスタンスメソッドだったら`#`を使う感じでお願いします。

Reply:
@komagata 
ご教示ありがとうございます。
承知しました 🙆